python/packages/ai/teams/auth/sso/sso_dialog.py

```diff
@@ -42,16 +44,15 @@ def __init__(self, name: str, options: SsoOptions, msal: ConfidentialClientAppli
         self._options = options
         self.initial_dialog_id = SSO_DIALOG_ID
 
-        # TODO - duplicate token exchange afterturn state 
-
+        # TODO - duplicate token exchange afterturn state
```

Comment:
do we need to remove this TODO?

Reply:
duplicate token exchange is handled on another PR, this TODO predated me. I think we are good to delete it as I can confirm I am actively working on that very file on this PR https://github.com/microsoft/teams-ai/pull/1890

will remove now.